### DISCUSSION TITLE
Support for Slide Subtitles in PowerPoint Conversion

### DISCUSSION THREAD

**Discussion by frettig-radiox** [jgm/pandoc#9564] [Ideas]
This feature request seeks to introduce subtitle support for PowerPoint slides. Similar to how Pandoc allows setting frame options for [Beamer slides (e.g., # Heading {frameoptions="option"})](https://pandoc.org/MANUAL.html#frame-attributes-in-beamer), it would be beneficial to enable subtitle addition on PowerPoint slides using a similar syntax, such as

 **# Slide1 {subtitle="This is a subtitle"}**

This addition would enhance the detail and clarity of PowerPoint presentations created from Markdown.

The ability to define these subtitles directly in the Markdown source would streamline the presentation creation process and align Pandoc's PowerPoint capabilities more closely with its Beamer support.

**Comment by hahntas** [comment]
This would be a great improvement.

**Comment by JulienStu** [comment]
Also need this, would be a great addition.
